python/taichi/examples/simulation/implicit_mass_spring.py

```diff
@@ -218,7 +237,7 @@ def displayGGUI(self, canvas, radius=0.01, color=(1.0, 1.0, 1.0)):
 
 
 def main():
-    ti.init(arch=ti.cpu)
+    ti.init(arch=ti.cpu, offline_cache=False)
```

Comment:
Is enabling `offline_cache` problematic?

Reply:
Nope, with `offline_cache` enabled, the example can run normally.

Reply:
Then we'd better not turn it off in the example :-)